mcs/class/corlib/Test/System.Reflection/AssemblyTest.cs

```diff
@@ -264,7 +264,7 @@ public void Corlib ()
 		public void Corlib_test ()
 		{
 			Assembly corlib_test = Assembly.GetExecutingAssembly ();
-#if MONODROID || FULL_AOT_DESKTOP || __WATCHOS__
+#if MONODROID || FULL_AOT_DESKTOP || __WATCHOS__ || MONO_SDK
```

Comment:
I am ok to go with MONOTOUCH for now

Reply:
That changes the test, xamarin.ios also defines it.

Reply:
Since you are adding `MONO_SDK` to the `monotouch` profile, using `MONOTOUCH` here has the same end result. And I concur with @marek-safar where we prefer to have `FEATURE_*` flags.